class4/js/calculadora.js

```diff
@@ -0,0 +1,54 @@
+let operacion = parseInt(prompt('OPERACIONES \n1 - Suma \n2 - Resta \n3 - Multiplicación \n4 - División'));
+
+switch (operacion){
+    case 1:
+        Sumar();
```

Comment:
```suggestion
        sumar();
```
Es más normal encontrar los nombres de funciones con la primer letra en minúscula: conocido también como [camelcase](https://developer.mozilla.org/en-US/docs/Glossary/Camel_case)